docs/en/developer.md

```diff
@@ -95,6 +95,18 @@ Add the following configuration to `mysite/_config/saml.yml` (make sure to repla
 	    entityId: "https://<idp-domain>/adfs/services/trust"
 	    x509cert: "<path-to-adfs-cert>.pem"
 	    singleSignOnService: "https://<idp-domain>/adfs/ls/"
+	  Security:
+	    signatureAlgorithm: "http://www.w3.org/2001/04/xmldsig-more#rsa-sha256"
+
+### A note on signature algorithm config
+
+The signature algorithm must match the setting in the ADFS relying party trust
+configuration. For ADFS it's possible to downgrade the default from SHA-256 to
+SHA-1 (not recommended), but in that case the YAML configuration needs to be
```

Comment:
maybe this should read like "For ADFS it's possible to downgrade the default from SHA-256 to
 +SHA-1, but is not recommend. To do that, you can change YAML configuration ..."